src/h5web/visualizations/shared/utils.ts

```diff
@@ -180,9 +180,21 @@ export function getTickFormatter(
   };
 }
 
-export function assertStr(val: unknown): asserts val is string {
+export function assertDefined<T>(
+  val: T,
+  message = 'Expected some value'
+): asserts val is NonNullable<T> {
+  if (val === undefined) {
+    throw new TypeError(message);
+  }
+}
```

Comment:
For info, when calling `assertDefined` with a value of type `unknown`, the return will still be `unknown`.